cmd/tailscale/cli/network-lock.go

```diff
@@ -168,7 +170,7 @@ func runNetworkLockInit(ctx context.Context, args []string) error {
 			return err
 		}
 		disablementValues = append(disablementValues, tka.DisablementKDF(supportDisablement))
-		fmt.Println("A disablement secret for Tailscale support has been generated and will be transmitted to Tailscale upon initialization.")
+		fmt.Fprintln(&disablementSecretsResp, "A disablement secret for Tailscale support has been generated and will be transmitted to Tailscale upon initialization.")
```

Comment:
Since this is only printed after the init succeeds, should we handle this one differently? Maybe we change it to "has been transmitted to Tailscale"?

Reply:
Forgive this rando coming in with a comment but I was in the neighborhood and saw the PR! If you're printing "Initialization complete" on line 183 it might make sense to leave this as is, just for user continuity. I'm not 100% familiar with how NetworkLockInit works but it looks like the process isn't really complete until we make it through that call. Again, just a rando offering random input, feel free to ignore me haha!

Reply:
Thank you! I believe we are now printing this after Tailnet Lock initialization succeeds, so I agree with @Erisa that it would make sense to change this to "has been transmitted to Tailscale.

Reply:
Done!